lib/src/models/section.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2021, grabajuice. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+part of spotify.models;
+
+@JsonSerializable(createToJson: false)
+class Section extends Object {
+  Section();
+
+  factory Section.fromJson(Map<String, dynamic> json) =>
+      _$SectionFromJson(json);
+
+  /// The starting point (in seconds) of the section.
+  double start;
+
+  /// The duration (in seconds) of the section.
+  double duration;
+
+  /// The confidence, from 0.0 to 1.0, of the reliability of the section’s “designation”.
+  double confidence;
+
+  /// The overall loudness of the section in decibels (dB). Loudness values are useful for comparing relative loudness of sections within tracks.
+  double loudness;
+
+  /// The overall estimated tempo of the section in beats per minute (BPM). In musical terminology, tempo is the speed or pace of a given piece and derives directly from the average beat duration.
+  double tempo;
+
+  /// The confidence, from 0.0 to 1.0, of the reliability of the tempo. Some tracks contain tempo changes or sounds which don’t contain tempo (like pure speech) which would correspond to a low value in this field.
```

Comment:
I'd love to see the comments split up into multiple lines, when they exceed around 80 characters. Makes it easier to read :)
This goes for all the model files!